src/olympia/activity/utils.py

```diff
@@ -74,10 +79,22 @@ def _extra_email_reply_parse(self, email):
     def get_uuid(self):
         addresses = [to.get('EmailAddress', '')
                      for to in self.email.get('To', [])]
+        to_notifications_alias = False
         for address in addresses:
             if address.startswith(self.address_prefix):
                 # Strip everything between "reviewreply+" and the "@" sign.
                 return address[len(self.address_prefix):].split('@')[0]
+            elif address.startswith(NOTIFICATIONS_FROM_EMAIL):
```

Comment:
nit: it's a complete email address, so you can do `address == NOTIFICATIONS_FROM_EMAIL` (I presumed it would be without the domain name in the constant)